README.md

```diff
@@ -515,6 +515,25 @@ SmalltalkCISpec {
   }
 }
 ```
+The coverage service will default to [coveralls.io][coveralls] to keep backwards compatibility. But you can use:
+- `#coveralls` will use [coveralls.io][coveralls]
+- `#codecov` will use [codecov.io][codecov]
+- `#local` will not attempt to upload the coverage report to any service. Useful for local runs or to setup the upload as a separate step in the build.
```

Comment:
I think it's a good debugging feature, and without setting the environment variables with the right tokens it will fail anyway if you run it locally by mistake.